aws-bundle/LICENSE

```diff
@@ -219,7 +219,7 @@ License: The Apache Software License, Version 2.0 - http://www.apache.org/licens
 
 --------------------------------------------------------------------------------
 
-Group: io.netty  Name: netty-buffer  Version: 4.1.112.Final
+Group: io.netty  Name: netty-buffer  Version: 4.1.118.Final
```

Comment:
in https://github.com/apache/iceberg/blob/1.8.x/gradle/libs.versions.toml#L73 we're using 4.1.117.Final btw

Reply:
This PR is on `main`, #12365 is for `1.8.x` branch.